nyxx.interactions/lib/src/Models/Interaction.dart

```diff
@@ -63,8 +51,29 @@ class Interaction extends SnowflakeEntity implements Disposable {
     this.token = raw["token"] as String;
 
     this.version = raw["version"] as int;
+
+    this.name = raw["data"]["name"] as String;
+
+    this.args = _generateArgs(raw["data"] as Map<String, dynamic>);
```

Comment:
New line between each statement seems unnecessary

Reply:
removed the extra spaces